torch/utils/data/dataloader.py

```diff
@@ -9,11 +9,13 @@
 import itertools
 import warnings
 
+
 import multiprocessing as python_multiprocessing
 import torch
 import torch.multiprocessing as multiprocessing
 from torch._utils import ExceptionWrapper
 from torch._six import queue, string_classes
+import psutil
```

Comment:
Is `psutil` a new dependency of PyTorch?

Reply:
This is just for test.